### ISSUE TITLE
Synthetics to push all Heartbeat config options

### ISSUE THREAD

**Issue by paulb-elastic** [elastic/synthetics#655]
This is the Synthetics version of the Kibana UI https://github.com/elastic/kibana/issues/140864 issue, which is about config options that can be done in traditional `heartbeat.yml` today, that are not supported in the [Synthetics UI](https://github.com/elastic/kibana/issues/140864), or via `push`.

This issue covers the requirement to add additional config options via `push` (further to what was initially implemented in https://github.com/elastic/synthetics/pull/593).

Note that there are other specific issues covering:
* TLS option: https://github.com/elastic/synthetics/issues/599
* Multiple hosts: https://github.com/elastic/synthetics/issues/627

This issue should focus on the other [configuration options in Heartbeat](https://www.elastic.co/guide/en/beats/heartbeat/current/monitor-options.html)

**Comment by shahzad31** [comment]
Also needs to support `ssl.renegotiation` and

**Comment by dominiqueclarke** [comment]
`response.include_body_max_bytes` was also requested by a customer.

**Comment by dominiqueclarke** [comment]
I don't think this is needed, as synthetics currently already pushes all the configuration it's provided, and Kibana is what tells it if it's supported or not. @paulb-elastic keeping this open for now but can probably close.
